deploy/helm/azure-industrial-iot/README.md

```diff
@@ -490,6 +837,191 @@ azure:
   keyVault:
     uri: <KeyVaultURI>
 
+  signalR:
+    connectionString: <SignalRConnectionString>
+
   auth:
     required: false
 ```
+
+## Special Notes
+
+### Resource Requests And Limits
+
+Helm chart does not set any [resource requests or limits](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/)
+for Pod Containers. We recommend to set memory requests to at least `"64Mi"` and cpu requests to at least
+`"20m"`. If you set resource limits, be informed that some components might require up to `"256Mi"` of
+memory, such as `frontend`, `registry`, `publisher` and a few others.
```

Comment:
>frontend [](start = 17, length = 8)

I think we use the term "engineering tool" for "frontend" in other places in documentation, I have no clue if this is right or not, but we should be consequent through documentation with naming of our components.

Reply:
Good point, thanks. I will change it to engineering tool and `engTool` for configuration parameters. But I will keep `/frontend` as URL path base for engineering tool. Sound good ?

Reply:
Decided to use `engineeringTool` for configuration as well.